script/travis

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
I'm gonna be honest, this is for programmers so if they can't get usage info from the simple 2 if's they shouldn't be using it.  I'm on the fence about whether I wish to comply with this because the script is so simple.